src/MDMMotionCurve.m

```diff
@@ -31,3 +31,14 @@ MDMMotionCurve MDMMotionCurveFromTimingFunction(CAMediaTimingFunction *timingFun
   [timingFunction getControlPointAtIndex:2 values:pt2];
   return MDMMotionCurveMakeBezier(pt1[0], pt1[1], pt2[0], pt2[1]);
 }
+
+MDMMotionCurve MDMMotionCurveReversed(MDMMotionCurve timingCurve) {
+  MDMMotionCurve reversed = timingCurve;
+  if (timingCurve.type == MDMMotionCurveTypeBezier) {
+    reversed.data[0] = 1 - timingCurve.data[2];
+    reversed.data[1] = 1 - timingCurve.data[3];
+    reversed.data[2] = 1 - timingCurve.data[0];
+    reversed.data[3] = 1 - timingCurve.data[1];
+  }
```

Comment:
What other curve types are there? Maybe it should be called `MDMMotionBezierCurveReversed` if it only supports the `.bezier` type?

Reply:
Changed the API name.